### ISSUE TITLE
Link to MOAC is dead

### ISSUE THREAD

**Issue by jonassmedegaard** [beAWARE-project/ontology#1]
README.md contains a link to the MOAC ontology, but the link is dead.

**Comment by skontopo** [comment]
You are absolutely right, many thanks for pointing it out! We will wait for a couple of days to see if they restore the link to the MOAC ontology.

**Comment by skontopo** [comment]
Link restored. Closing issue.
